web/.husky/pre-commit

```diff
@@ -17,6 +17,11 @@
 
 #!/usr/bin/env sh
 
+# Git commit does check when web module does not install dependencies.
```

Comment:
Could we remove this precommit check? IMO, the CI would check the frontend code style and the contribution guide introduces the way to reformat frontend code.

Reply:
ok.